README.md

```diff
@@ -72,3 +75,168 @@ The development status of a plugin used to be recorded explicitly on the plugin
 Over time, we've moved closer and closer to adopting the [development status trove classifer](https://pypi.org/classifiers/), so we now suggest to just use those in your `setup.json`/`setup.cfg`/... file of your plugin.
 
 If no development status is specified, the status will default to 'planning'.
+
+## How to fix registry warnings and errors
```

Comment:
I get that installing the registry locally is useful for debugging, but I think our goal should be that plugin developers do _not_ have to do this.

Once https://github.com/aiidateam/aiida-registry/issues/272 is fixed, I think the default suggestion should be to open a PR and look at the warnings in the build.
Is there an intermediate suggestion we can write in the README until we have that capability?

As for the rest of this section, in order to keep the README short, I suggest moving it in a separate markdown document 'debugging.md' and link to it from the README.

Reply:
> I get that installing the registry locally is useful for debugging, but I think our goal should be that plugin developers do not have to do this.

I agree. It is better to move to the repository wiki and link from README.